app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee.erb

```diff
@@ -100,6 +101,19 @@ ShortFormApplicationService = ($localStorage) ->
   Service.cancelHouseholdMember = ->
     householdMembers = Service.householdMembers.filter (m) ->
       (m != Service.householdMember && m.id != Service.householdMember.id)
+    # clear out preferences where this householdMember was selected
```

Comment:
@kaptron I always wondered if namespacing would work or if we should nest the data structure.

something like 

`application:  { preferences: { preferenceName: [householdmembers] }` 

or something. it takes some work though and not sure if it's worth the work if these are the only preferences we'll be working with.